### ISSUE TITLE
Grid: sort arrows missing text equivalent (1.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#939]
### Expected behavior
Arrows visually indicate that the column is currently sorted and in what order. This information is also available in text form equal to the visual representation. 

### Actual behavior
There is no text equivalent. It's only available to users who can see the arrows.
